### ISSUE TITLE
streaming_load: Failed to read a compressed parquet file. Use the cargo feature "io_parquet_compression" to read compressed parquet files

### ISSUE THREAD

**Issue by BohuTANG** [datafuselabs/databend#4499]
**Summary**

### Release
curl -LJO https://github.com/datafuselabs/databend/releases/download/v0.6.95-nightly/databend-v0.6.95-nightly-x86_64-unknown-linux-gnu.tar.gz

### Parquet
https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.parquet

Export by clickhouse:
```
select * from books into outfile 'books.parquet' format PARQUET;
```

### How to reproduce(Parquet)
https://databend-git-fork-bohutang-doc-parquet-databend.vercel.app/doc/load-data/local

```
echo curl -H \"insert_sql:insert into book_db.books format Parquet\" -H \"skip_header:1\" -F  \"upload=@./books.parquet\" -XPUT http://127.0.0.1:8081/v1/streaming_load|bash

Code: 1024, displayText = External format error: Failed to read a compressed parquet file. Use the cargo feature "io_parquet_compression" to read compressed parquet files.
```

**Comment by BohuTANG** [comment]
Should enable the io_parquet_compression feature:

`arrow = { package = "arrow2", git = "https://github.com/datafuse-extras/arrow2", default-features = false, features = ["io_parquet", "io_parquet_compression"], rev = "b0774c8" }`

**Comment by BohuTANG** [comment]
Fixed in #4500
